pkg/pillar/dpcreconciler/linuxitems/adapter.go

```diff
@@ -195,12 +195,21 @@ func (c *AdapterConfigurator) Create(ctx context.Context, item depgraph.Item) er
 		c.Log.Error(err)
 		return err
 	}
+	// Make sure that corresponding bridge and interface are UP right
+	// after MAC address assignment and interface rename. Order matters,
+	// otherwise host returns `Device or resource busy`.
 	if err := netlink.LinkSetUp(bridge); err != nil {
 		err = fmt.Errorf("netlink.LinkSetUp(%s) failed: %v",
 			adapter.IfName, err)
 		c.Log.Error(err)
 		return err
 	}
+	if err := netlink.LinkSetUp(kernLink); err != nil {
+		err = fmt.Errorf("netlink.LinkSetUp(%s) failed: %v",
+			adapter.IfName, err)
```

Comment:
Should be `kernIfname` in the error message.

Reply:
Yep, thanks. Done.